---
float32 current_position
int8 success
int8 blocked
int8 endstop
int8 status
int8 referenced